lib/spoom/cli/bump.rb

```diff
@@ -95,9 +97,13 @@ def bump(directory = ".")
 
         undo_changes(files_with_errors, from)
 
+        if options[:count_errors]
+          say("Errors: #{errors.length}")
+        end
+
         files_changed = files_to_bump - files_with_errors
         print_changes(files_changed, command: cmd, from: from, to: to, dry: dry, path: exec_path)
-        undo_changes(files_to_bump, from) if dry
+        undo_changes(files_to_bump, from) if dry || options[:count_errors]
```

Comment:
Why do we need this?

Reply:
In my opinion, it would be surprising for this option to also bump the files. At least I'd expect `--count-errors` to just count errors and not attempt to keep the files bumped.

Do you feel like the other way around is more intuitive?

Reply:
In that case, can we make it so you have to pass both `dry` and `count-errors`?

Like 

```
$ spoom bump --count-errors
Error: `--count-errors` can only be used with `--dry`.
```

?

Reply:
Yeah, that makes more sense. 👍